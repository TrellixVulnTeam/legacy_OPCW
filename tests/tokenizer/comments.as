# Hello; I'm a comment
32 # Another comment
