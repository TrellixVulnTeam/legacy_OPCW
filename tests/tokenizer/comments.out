<decimal integer> '32'
<end>
